src/nni_manager/core/nnimanager.ts

```diff
@@ -148,6 +148,7 @@ class NNIManager implements Manager {
     }
 
     public async resumeExperiment(): Promise<void> {
+        this.log.info(`Resuming experiment: ${this.experimentProfile.id}`);
```

Comment:
I suggest we go through other places to make sure all the logs follow correct log level.

Reply:
Gone through training services and database, more log updated.